Sources/Accord.Imaging.WPF/Accord.Imaging.WPF.csproj

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Hi @heltonbiker, 

Please start with a real copy of the Accord.Imaging .csproj file, and change the only the AssemblyName, RootNamespace (and maybe XML documentation) fields. This will make sure all the build actions, target directories and documentation paths are consistent with the rest of the framework.